packages/api/prisma/schema.prisma

```diff
@@ -14,33 +14,62 @@ datasource db {
 }
 
 model Book {
-  id      Int       @id
-  name    String
-  search  String    @unique // This gives us a human readable key to look up books from URL params.
-  Chapter Chapter[]
+  id     Int     @id
+  name   String
+  search String  @unique // This gives us a human readable key to look up books from URL params.
+  verses Verse[]
 }
 
-model Chapter {
-  id     Int     @id
-  number Int
-  book   Book    @relation(fields: [bookId], references: [id])
-  bookId Int
-  Verse  Verse[]
+model Verse {
+  id           Int                @id
+  number       Int
+  chapter      Int
+  book         Book               @relation(fields: [bookId], references: [id])
+  bookId       Int
+  words        Word[]
+  translations VerseTranslation[]
 
-  @@unique([bookId, number])
+  @@unique([bookId, chapter, number])
 }
 
-model Verse {
-  id        Int     @id
-  number    Int
-  chapter   Chapter @relation(fields: [chapterId], references: [id])
-  chapterId Int
+model Word {
+  id      String    @id
+  text    String
+  verse   Verse     @relation(fields: [verseId], references: [id])
+  verseId Int
+  order   Int
+  form    LemmaForm @relation(fields: [formId], references: [id])
+  formId  String
+
+  @@unique([verseId, order])
+}
+
+model LemmaForm {
+  id      String @id
+  grammar String
+  lemma   Lemma  @relation(fields: [lemmaId], references: [id])
+  lemmaId String
+  word    Word[]
+}
+
+model Lemma {
+  id    String      @id
+  forms LemmaForm[]
+}
+
+model VerseTranslation {
+  verse       Verse    @relation(fields: [verseId], references: [id])
+  verseId     Int
+  language    Language @relation(fields: [languageId], references: [id])
+  languageId  String   @db.Uuid
+  translation String
 
-  @@unique([chapterId, number])
+  @@id([verseId, languageId])
```

Comment:
A verse translation is connected to the verse and the language and provides the natural language translation